### PR TITLE
Everybody Needs a Kiss

### PR DESCRIPTION
**Pull request by neuPanda**
Description.
added kiss interaction and texts
added lick interaction and texts

:cl:
- add: Kiss an Lick

### CODE REVIEW

File: Resources/Prototypes/Floof/Interactions/noop_interactions.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+- type: Interaction
+  id: Kiss
+  parent: [BaseGlobal, BaseHands]
+  priority: 2
+  #icon: /Textures/Interface/Actions/hug.png
```
Comment (Mnemotechnician) [inline Resources/Prototypes/Floof/Interactions/noop_interactions.yml:5]:
```suggestion
  priority: 0
```

File: Resources/Prototypes/Floof/Interactions/noop_interactions.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  delay: 0.7
+  range: {max: 1}
+  hideByRequirement: true
+  requirement:
+    !type:MobStateRequirement
+    inverted: true
```
Comment (Mnemotechnician) [inline Resources/Prototypes/Floof/Interactions/noop_interactions.yml:11]: You don't want this to be usable on dead bodies. Probably. 
```suggestion
    allowedStates: [Alive, Critical]
```

**Review by Mnemotechnician — CHANGES_REQUESTED** [review]
Same applies to the second action-

File: Resources/Locale/en-US/floofstation/interaction/verbs/noop.ftl [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+interaction-Kiss-description = A kiss melts the pains away.
+interaction-Kiss-success-self-popup = You kiss {THE($target)}.
+interaction-Kiss-success-target-popup = {THE($user)} Kisss you.
+interaction-Kiss-success-others-popup = {THE($user)} Kisss {THE($target)}.
+
+interaction-Lick-name = Lick
+interaction-Lick-description = Lick your co-worker, what HR?.
+interaction-Lick-success-self-popup = You lick {THE($target)}.
+interaction-Lick-success-target-popup = {THE($user)} Licks you.
+interaction-Lick-success-others-popup = {THE($user)} Licks {THE($target)}.
```
Comment (Mnemotechnician) [inline Resources/Locale/en-US/floofstation/interaction/verbs/noop.ftl:11]: Those need to be lowercased
```suggestion
interaction-Kiss-success-target-popup = {THE($user)} kisses you.
interaction-Kiss-success-others-popup = {THE($user)} kisses {THE($target)}.

interaction-Lick-name = Lick
interaction-Lick-description = Lick your co-worker, what HR?.
interaction-Lick-success-self-popup = You lick {THE($target)}.
interaction-Lick-success-target-popup = {THE($user)} licks you.
interaction-Lick-success-others-popup = {THE($user)} licks {THE($target)}.
```

File: Resources/Prototypes/Floof/Interactions/noop_interactions.yml (hunk 2 above)
Comment (neuPanda) [reply Resources/Prototypes/Floof/Interactions/noop_interactions.yml:10]: :( no, there are people who are going to want that

**Comment by Mnemotechnician** [comment]
https://github.com/Simple-Station/Einstein-Engines/pull/823 has been merged and is going to be included in #178, so uh please update those later to add some kinda moodlet.

**Comment by Fansana** [comment]
#178 is added please revisit this

**Comment by neuPanda** [comment]
i fuckin hate yml

**Comment by Memeji** [comment]
What is the current status for this PR?

**Review by ShatteredSwords — COMMENTED** [review]
The kiss and lick make a hug sound. If this is a limitation or if it is hard to find these, that is totally fair. If you think you can find a properly licensed sound for each of these actions though, that'd be cool!

**Comment by neuPanda** [comment]
> The kiss and lick make a hug sound. If this is a limitation or if it is hard to find these, that is totally fair. If you think you can find a properly licensed sound for each of these actions though, that'd be cool!

not sure where the sounds are or how they work,  but i am sure there is something out there.
that said you also got me to go back and look at it and realize i missed a 2 lines in the copy paste that needed to be updated

**Comment by ShatteredSwords** [comment]
> not sure where the sounds are or how they work, but i am sure there is something out there. that said you also got me to go back and look at it and realize i missed a 2 lines in the copy paste that needed to be updated

Base interactions are located [here](https://github.com/Fansana/floofstation1/blob/master/Resources/Prototypes/Interactions/base.yml), which is what looks like ties the sound to whatever. Looks like you'd have to make a new base interaction within the Floof folder and parent it to that. Completely up to you of course! I think we could do it in the future just as easily!

**Comment by Mnemotechnician** [comment]
There's no need to create new bases unless you intend to re-use them

**Comment by neuPanda** [comment]
I need to run everything again after the changes i did yesterday just have not had a chance to open VS yet.  it was just an issue i saw in the PR, so i did the quick fix remotely and do still intend to test it

**Review by FoxxoTrystan — APPROVED** [review]

**Comment by Memeji** [comment]
> I need to run everything again after the changes i did yesterday just have not had a chance to open VS yet. it was just an issue i saw in the PR, so i did the quick fix remotely and do still intend to test it

Any update on this?

**Comment by neuPanda** [comment]
audio's have been updated

**Comment by ShatteredSwords** [comment]
I will test this when I have the chance, very excited!

**Review by ShatteredSwords — COMMENTED** [review]
Sounds don't seem to be working, still uses the default sounds after playtesting. Not sure why! Everything else still is good though.

File: Resources/Prototypes/Floof/Interactions/base.yml [hunk 4]
```diff
@@ -0,0 +1,29 @@
+# Base interaction that involves hands
```
Comment (Mnemotechnician) [inline Resources/Prototypes/Floof/Interactions/base.yml:1]: This entire file can be removed in favor of the suggestion below

File: Resources/Prototypes/Floof/Interactions/mood_interactions.yml [hunk 5]
```diff
@@ -0,0 +1,31 @@
+- type: Interaction
+  id: Kiss
+  parent: [BaseGlobal, BaseLips]
+  priority: 0
+  delay: 0.4
+  range: {max: 1}
+  hideByRequirement: true
+  requirement:
+    !type:MobStateRequirement
+    inverted: true
```
Comment (Mnemotechnician) [inline Resources/Prototypes/Floof/Interactions/mood_interactions.yml:10]: Lesser priority, inlined effect
```suggestion
  parent: [BaseGlobal, BaseHands]
  priority: -1
  delay: 0.4
  range: {max: 1}
  hideByRequirement: true
  requirement:
    !type:MobStateRequirement
    inverted: true
  effectSuccess:
    popup: Obvious
    sound: {path: /Audio/Floof/Lewd/kiss.ogg}
```

File: Resources/Prototypes/Floof/Interactions/mood_interactions.yml [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+  id: Lick
+  parent: [BaseGlobal, BaseTongue]
+  priority: 0
+  #icon: /Textures/Interface/Actions/hug.png
+  delay: 0.7
+  range: {max: 1}
+  hideByRequirement: true
+  requirement:
+    !type:MobStateRequirement
+    inverted: true
```
Comment (Mnemotechnician) [inline Resources/Prototypes/Floof/Interactions/mood_interactions.yml:26]: Similarly,
```suggestion
  parent: [BaseGlobal, BaseTongue]
  priority: -2
  #icon: /Textures/Interface/Actions/hug.png
  delay: 0.7
  range: {max: 1}
  hideByRequirement: true
  effectSuccess:
    popup: Obvious
    sound: {path: /Audio/Floof/Lewd/lick.ogg}
  requirement:
    !type:MobStateRequirement
    inverted: true
```

**Review by FoxxoTrystan — APPROVED** [review]

File: Resources/Prototypes/Floof/Interactions/base.yml (hunk 4 above)
Comment (neuPanda) [reply Resources/Prototypes/Floof/Interactions/base.yml:1]: will look into this when i have some time,  been a bit busty and have not had time to look into this yet.  sorry for the delay

**Comment by Memeji** [comment]
Were the above suggestions / conversations resolved?

**Comment by neuPanda** [comment]
> Were the above suggestions / conversations resolved?

not yet.  IRL has been hitting me hard.  I have been more focused on how to come up with 30K for how to repair the house than i am the correct way to kiss somebody

**Comment by Memeji** [comment]
> > Were the above suggestions / conversations resolved?
> 
> not yet. IRL has been hitting me hard. I have been more focused on how to come up with 30K for how to repair the house than i am the correct way to kiss somebody

Take the time you need. Apologies if I sound like I'm rushing, I don't intend to do so, just see where the PR stands at the moment. Focus on the more important things there, hoping all the best.

**Comment by neuPanda** [comment]
> Take the time you need. Apologies if I sound like I'm rushing, I don't intend to do so, just see where the PR stands at the moment. Focus on the more important things there, hoping all the best.

no worries, i think i may have sounded more aggressive than i was, i didn't feel like you were rushing me, I'm just ready to throw hands at the world.  unless shit hits the fan again today, i want to get this cleaned up when i get off work today so it can be in for the weekend

**Comment by FoxxoTrystan** [comment]
Why this was merged without approval?
Aaa

**Pull request merged**